### PR TITLE
Fixed the issue in deleting sample apps if subscription has disabled

### PR DESCRIPTION
**Pull request by lakshani**

### CODE REVIEW

File: components/org.wso2.carbon.appmgt.sample.deployer/src/main/java/org/wso2/carbon/appmgt/sample/deployer/deploy/Deployer.java [hunk 1]
```diff
@@ -128,4 +136,18 @@ public boolean deploySample(NativeObject request) throws AppManagementException
         }
         return response;
     }
+
+    private boolean isSubscriptionEnabled() {
+        AppManagerConfiguration appManagerConfiguration = ServiceReferenceHolder.getInstance()
+                .getAPIManagerConfigurationService().getAPIManagerConfiguration();
+        Boolean isSubscriptionEnabled = false;
```
Comment (dinusha-dilrukshi) [inline components/org.wso2.carbon.appmgt.sample.deployer/src/main/java/org/wso2/carbon/appmgt/sample/deployer/deploy/Deployer.java:143]: No need of using Boolean object here. Use primitive 'boolean'.

File: components/org.wso2.carbon.appmgt.sample.deployer/src/main/java/org/wso2/carbon/appmgt/sample/deployer/deploy/Deployer.java [hunk 2]
```diff
@@ -63,18 +66,21 @@ public boolean deploySample(NativeObject request) throws AppManagementException
             log.error("Error while parsing JSON object", e);
             throw new AppManagementException("Error while parsing JSON object", e);
         }
-        UserAdminServiceClient userAdminServiceClient;
-        try {
-            userAdminServiceClient = new UserAdminServiceClient();
-            userAdminServiceClient.addUser("subscriber_" + username);
-        } catch (UserAdminUserAdminException e) {
+        Boolean hasSubscriptionEnabled = isSubscriptionEnabled();
```
Comment (dinusha-dilrukshi) [inline components/org.wso2.carbon.appmgt.sample.deployer/src/main/java/org/wso2/carbon/appmgt/sample/deployer/deploy/Deployer.java:69]: Can use primitive boolean.

Comment (lakshani) [reply components/org.wso2.carbon.appmgt.sample.deployer/src/main/java/org/wso2/carbon/appmgt/sample/deployer/deploy/Deployer.java:69]: This line has removed from latest code.

File: components/org.wso2.carbon.appmgt.sample.deployer/src/main/java/org/wso2/carbon/appmgt/sample/deployer/deploy/Deployer.java (hunk 1 above)
Comment (lakshani) [reply components/org.wso2.carbon.appmgt.sample.deployer/src/main/java/org/wso2/carbon/appmgt/sample/deployer/deploy/Deployer.java:143]: Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/765a403f533c64e20704c6b0424d16f7acee08d0

File: components/org.wso2.carbon.appmgt.sample.deployer/src/main/java/org/wso2/carbon/appmgt/sample/deployer/appcontroller/ProxyApplicationCreator.java [hunk 3]
```diff
@@ -94,7 +97,8 @@ public ProxyApplicationCreator() throws AppManagementException {
      *                                Throws this when policy id is failed while requesting
      *                                Throws this when failed to create,publish or subscribe web application
      */
```
Comment (lakshani) [inline components/org.wso2.carbon.appmgt.sample.deployer/src/main/java/org/wso2/carbon/appmgt/sample/deployer/appcontroller/ProxyApplicationCreator.java:99]: Java doc comment issue has fixed in https://github.com/wso2/carbon-appmgt/pull/539/files

**Pull request merged**